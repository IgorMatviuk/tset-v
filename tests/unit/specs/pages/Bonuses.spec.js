import { shallowMount } from '@vue/test-utils'

import BonusesPage from '@/pages/-bonuses'

describe('BonusesPage', () => {
  test('create bonuses page', () => {
    const wrapper = shallowMount(BonusesPage, {
      mocks: {
        $t: jest.fn(),
        $i18n: {
          path: jest.fn()
        }
      }
    })

    expect(wrapper.element).toMatchSnapshot()
  })
})
