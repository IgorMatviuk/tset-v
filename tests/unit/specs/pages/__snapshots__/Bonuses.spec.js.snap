// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BonusesPage create bonuses page 1`] = `
<main
  class="bonuses-page"
>
  <thebreadcrumb-stub
    isfloat="true"
    items="[object Object],[object Object]"
  />
   
  <blockjumbotron-stub
    class="bonuses-page__jumbotron"
    content="[object Object]"
  >
    <div
      class="bonuses-page__title"
    >
      
    </div>
     
    <div
      class="bonuses-page__subtitle"
    >
      Virna 
      <b>
        points
      </b>
    </div>
  </blockjumbotron-stub>
   
  <blockbonusescontent-stub
    content=<h2>уважаемые покупатели!</h2>
<p>Специально для Вас мы разработали программу лояльности, которая сделает покупки в нашем интернет-магазине более приятными и выгодными. С помощью нашей программы лояльности, Вы сможете экономить на покупках до 50% от стоимости товаров. Начисление и списание бонусов происходит даже при покупке подарочных сертификатов и вещей со скидкой. </p>
  />
  <blockbonusesmemberroad-stub
    content="[object Object]"
  />
  <blockbonusescontent-stub
    content=<h2>что такое бонусы?</h2>
<p>Бонусы – это денежное вознаграждение за то, что вы покупаете в интернет-магазине Virna. За каждую покупку на ваш счет будут начислены бонусы. Бонусы будут называться Virna Points.</p>
<p>В нашей бонусной программе 1 бонус = 1 гривне, и потратить эти бонусы можно на любые последующие покупки на нашем сайте.</p>
  />
  <blockblockquote-stub
    class="bonuses-page__blockquote"
    content="Бонусы позволяют <b>экономить деньги</b> на покупках! Оплачивайте бонусами <b>до 50% от суммы заказа</b>"
  />
  <blockbonusescontent-stub
    content=<h3>Как стать участником программы:</h3>
<p>Каждому Пользователю при регистрации на сайте virna.ua открывается бонусный счет, и он автоматически становится участником программы лояльности «Virna Points». Став участником дисконтной программы, Вы накапливаете бонусы, которые можете использовать для получения скидок при оформлении заказов.</p>
<p>Каждый раз, когда Вы заходите на сайт и оплачиваете покупки - Вы получаете бонусы. Бонус за покупку начисляется от суммы оплаченного Вами заказа, а его процент зависит от суммы всех предыдущих покупок. Более детально в таблице.</p>
  />
  <blockbonusestable-stub
    items="[object Object],[object Object],[object Object]"
  />
  <blockbonusescontent-stub
    content=<p>После оплаты покупки, Вы можете посмотреть сумму бонусов и общую сумму покупок в личном кабинете.</p>
  />
  <blockbonusescontent-stub
    content=<h3>Как проверить состояние бонусного счета?</h3>
<p>Все ваши бонусы хранятся на отдельном виртуальном счету. Состояние вашего бонусного счета и все операции по начислению и списанию бонусов вы можете всегда посмотреть в личном кабинете в закладке «Мои бонусы». Бонусы доступны к использованию через 14 дней после получения и оплаты заказа. Оплатить бонусами можно не более 50% стоимости заказа.</p>
  />
  <blockbonusescontent-stub
    content=<h3>Как использовать бонусы?</h3>
<p>Для использования бонусов необходимо авторизоваться на сайте, и если на вашем счету имеются неиспользованные бонусы, то при оформлении заказа, система автоматически покажет вам количество имеющихся бонусов и предложит использовать их для оплаты текущего заказа, при этом Вам будет начислены бонусы. Баллы начисляется на ту часть товара, которая оплачена деньгами (если частично использованы баллы). Бонусы, начисленные за оплаченные Вами товары, можно потратить на любые последующие покупки в нашем сайте.</p>
  />
  <blockblockquote-stub
    class="bonuses-page__blockquote"
    content="Курс обмена бонусов <b>1 бонус = 1 гривна.</b> Чем больше покупка, <b>тем больше экономия!</b>"
  />
  <blockbonusesvip-stub
    content="[object Object]"
  />
  <blockbonusesfaq-stub
    content="[object Object]"
  />
</main>
`;
